tools/check-setuppy

```diff
@@ -0,0 +1,89 @@
+#! /usr/bin/env python3
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2018, Arm Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import subprocess
+import sys
+
+LISA_HOME = os.getenv('LISA_HOME', '.')
+
+def call_git(*args):
+    return subprocess.check_output(('git', *args)).decode('utf-8')
+
+def get_git_sha1(ref):
+    return call_git('rev-parse', ref).strip()
+
+def main():
+    parser = argparse.ArgumentParser(description='''
```

Comment:
We use `textwrap.dedent()` in `TestEnv.from_cli()`, *I think* because otherwise the indentation gets included.